grafana/grafana.spec

```diff
@@ -4,17 +4,22 @@
 %global version 6.2.2
 
 Name:             grafana
-Version:          %{version}
-Release:          3%{?dist}
+Version:          6.2.2
+Release:          4%{?dist}
 Summary:          Metrics dashboard and graph editor
 License:          ASL 2.0
 URL:              https://grafana.org
 
+%global webpack_hash 5b248f47a09de006a30a27901eaa5471e5660a86d52ea601a9be0d6c8a9393f2
+
 # Source0 contains the tagged upstream sources
-Source0:          https://github.com/grafana/grafana/archive/v%{version}.tar.gz
+Source0:          https://github.com/grafana/grafana/archive/v%{version}/grafana-%{version}.tar.gz
 
 # Source1 contains the front-end javascript modules bundled into a webpack
-Source1:          grafana_webpack-%{version}.tar.gz
+Source1:          grafana_webpack-%{version}.%{webpack_hash}.tar.gz
+
+# Source2 is the script to create the above webpack from grafana sources
```

Comment:
Why was this added? Do we need the commented code?

Reply:
To match the product content and make it easier to compare